bssl-compat/source/SSL_CTX_set_compliance_policy.cc

```diff
@@ -111,8 +110,8 @@ static int Configure(SSL_CTX *ctx) {
       // Encrypt-then-MAC extension is required for all CBC cipher suites and so
       // it's easier to drop them.
       SSL_CTX_set_strict_cipher_list(ctx, kTLS12Ciphers) &&
-      SSL_CTX_set1_group_ids(ctx, kGroups, OPENSSL_ARRAY_SIZE(kGroups)) &&
-      SSL_CTX_set_signing_algorithm_prefs(ctx, kSigAlgs) &&
+      ossl.ossl_SSL_CTX_set1_groups (ctx, kGroups, OPENSSL_ARRAY_SIZE(kGroups)) &&
+      ossl.ossl_SSL_CTX_set1_sigalgs_list(ctx, kSigAlgs) &&
       SSL_CTX_set_verify_algorithm_prefs(ctx, kSigAlgs);
```

Comment:
Remove this call to `SSL_CTX_set_verify_algorithm_prefs()`.

Even if we _could_ implement `SSL_CTX_set_verify_algorithm_prefs()` (we can't) this call is logically redundant because the _same_ list of sigalgs is also passed to `SSL_CTX_set1_sigalgs_list()` which is what gets used if the `signature_algorithms_cert` extension is not present.